### DISCUSSION TITLE
diff 着色工具：delta

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#71] [3.分享创造]
https://github.com/dandavison/delta  比 https://github.com/so-fancy/diff-so-fancy 好用。delta 是基于语法的 diff，diff 结果很精准，阅读起来比较轻松。而且它支持 side-by-side diff，还支持 blame, grep。很有用。

官方仓库提供的配色不好看，我自己写了一个：https://github.com/adoyle-h/delta-theme-adoyle-dark

### side-by-side=true

```gitconfig
[delta]
    navigate = true
    features = adoyle-dark
    side-by-side = true
```

![2024-09-16_21 20 53@2x](https://github.com/user-attachments/assets/c3704471-fd34-427a-935b-dccd869ffc14)

### side-by-side=false

```gitconfig
[delta]
    navigate = true
    features = adoyle-dark
    side-by-side = false
```

![2024-09-16_21 53 10@2x](https://github.com/user-attachments/assets/680ff099-0300-4307-9e37-da48e9916bb0)

### git blame

![2024-09-16_21 55 58@2x](https://github.com/user-attachments/assets/892b80ca-f209-44fa-a65e-7880e306b5ee)

### git grep

![2024-09-16_22 00 40@2x](https://github.com/user-attachments/assets/2dfca169-4054-448e-936c-71baaceacbef)

**Comment by kalxd** [comment]
不放张截图吗？

**Reply by adoyle-h** [reply]
好，更新截图了
